### DISCUSSION TITLE
Avalonia Plugins can't be installed in Visual Studio2022?

### DISCUSSION THREAD

**Discussion by BugsLifeooo** [AvaloniaUI/Avalonia#12496] [Q&A]
My VisualStudio2022 is version 17.4, but it runs in an Intranet environment. I imported a plugin for Avalonia2022, version 0.10.18.4, which cannot be installed in visual studio2022. The message "Failed to establish trust relationship for ssl/tls secure channel." is displayed. What should I do?

**Comment by Flithor** [comment]
Maybe? 
https://github.com/NuGet/NuGetGallery/issues/7705
